### PR TITLE
Add all tailable cursor tests

### LINKED ISSUE #605: Enable all tests for tailable cursors
We should enable all tests for tailable cursor we can find. As much as possible should pass when run against MongoDB.

### PR DESCRIPTION
**Pull request by b1ron**
Closes #605.

### CODE REVIEW

**Review by AlekSi — APPROVED** [review]

File: tests/mongo-go-driver.yml [hunk 1]
```diff
@@ -129,9 +131,9 @@ results:
 
   postgresql:
     stats:
-      fail: 10
+      fail: 13
       skip: 19
-      pass: 633
+      pass: 630
```
Comment (AlekSi) [inline tests/mongo-go-driver.yml:136]: We should not ignore this number going down

File: tests/mongo-go-driver.yml [hunk 2]
```diff
@@ -36,6 +36,11 @@ results:
 
       # error terminating open transactions: (CommandNotFound) no such command: 'killAllSessions'
       - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec
+      
+      # https://github.com/FerretDB/FerretDB/issues/3601
```
Comment (AlekSi) [inline tests/mongo-go-driver.yml:40]: That issue is unrelated to that apparent regression

Comment (b1ron) [reply tests/mongo-go-driver.yml:40]: Ok, I will revert the changes and investigate the issue separately.

**Comment by b1ron** [comment]
@AlekSi I tidied up `mongo.yml` config file with updated links etc.

File: tests/mongo.yml [hunk 3]
```diff
@@ -56,26 +56,35 @@ args:
...
+  - mongo/jstests/core/query/cursor/tailable_cursor_invalidation.js
+  - mongo/jstests/core/query/cursor/tailable_getmore_batch_size.js
+  - mongo/jstests/core/query/cursor/tailable_skip_limit.js
+  - mongo/jstests/core/write/crud_api.js
 
 results:
   includes:
     aggregation:
-      - mongo/jstests/aggregation/ifnull.js # unhandled element type `Undefined (value) Deprecated`
+      - mongo/jstests/aggregation/ifnull.js # https://github.com/FerretDB/FerretDB/issues/2286
```
Comment (AlekSi) [inline tests/mongo.yml:72]:
```suggestion
      # https://github.com/FerretDB/FerretDB/issues/2286
      - mongo/jstests/aggregation/ifnull.js
```

To not use two comment styles at the same time
